src/Umbraco.Web.BackOffice/Controllers/BlockGridSampleHelper.cs

```diff
@@ -0,0 +1,192 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Umbraco.Cms.Core;
+using Umbraco.Cms.Core.Models;
+using Umbraco.Cms.Core.Models.ContentEditing;
+using Umbraco.Cms.Core.PropertyEditors;
+using Umbraco.Cms.Core.Security;
+using Umbraco.Cms.Core.Services;
+using Umbraco.Cms.Web.Common.ActionsResults;
+using Umbraco.Extensions;
+
+namespace Umbraco.Cms.Web.BackOffice.Controllers;
```

Comment:
This is nitpicky, but it should probably live in its own folder

Reply:
@Zeegaan I wanted to, but there is no "Helpers" folder or namespace. I put it here on purpose, because there was another helper implementation already 😆 

![image](https://user-images.githubusercontent.com/7405322/193819859-188a2564-4343-4599-a716-2249652a0ab8.png)